### PR TITLE
[test] Add test for issue5781

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Add test for issue #5781
- This case is passed on v0.21.2 and v0.27.4 after adding "nwfaketop"

### CODE REVIEW

File: test/sanity/issue5781-X-Frame-Options/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name":"issue5781-X-Frame-Options",
+  "main":"index.html",
+  "node-remote": "*://*/*"
```
Comment (rogerwang) [inline test/sanity/issue5781-X-Frame-Options/package.json]: Is this line necessary for this case?

Comment (wanghongjuan) [reply test/sanity/issue5781-X-Frame-Options/package.json]: No, just removed it.

File: test/sanity/issue5781-X-Frame-Options/test.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(5)
+time.sleep(1)
+try:
+    print driver.current_url
+    print 'waiting to load https://web.whatsapp.com/'
+    wait_for_element_tag(driver, "iframe")
+    assert(driver.find_element_by_tag_name("iframe") is not None)
+    driver.switch_to.frame(driver.find_element_by_tag_name("iframe"))
+    result = wait_for_element_class(driver, '_2BXqy')
```
Comment (rogerwang) [inline test/sanity/issue5781-X-Frame-Options/test.py]: This depends on the design of remote site, which could be changed and break our case in future.

Comment (wanghongjuan) [reply test/sanity/issue5781-X-Frame-Options/test.py]: Following we talked before, created a local http server instead of this remote site.

**Comment by rogerwang** [comment]
Thanks. Does it fail (or is it effective) when 'nwfaketop' is not used?

**Comment by wanghongjuan** [comment]
It is failed without “nwfaketop”.

**Pull request merged**